lib/fiware-iotagent-lib.js

```diff
@@ -156,10 +175,6 @@ function doActivate(newConfig, callback) {
         ], callback);
     };
 
-    if (!config.getConfig().dieOnUnexpectedError) {
```

Comment:
Why delete this check? Apparently is not related with current feature, is it?

Reply:
it was moved upfront